src/redux/slices/flattened-sublayers-slice.tsx

```diff
@@ -4,26 +4,36 @@ import {
   Sublayer,
   TilesetType,
   TilesetMetadata,
+  ComparisonSideMode,
 } from "../../types";
 import { I3SBuildingSceneLayerLoader } from "@loaders.gl/i3s";
 import { buildSublayersTree } from "../../utils/sublayers";
 import { load } from "@loaders.gl/core";
 import { RootState } from "../store";
 
-// Define a type for the slice state
-interface flattenedSublayersState {
+// Define a type for the slice states
+type sublayersState = {
```

Comment:
Types must start from upper case
```suggestion
type SublayersState = {
```

Reply:
Done